language_service/src/test_utils.rs

```diff
@@ -48,10 +48,10 @@ pub(crate) fn compile_with_fake_stdlib(source_name: &str, source_contents: &str)
             "<std>".into(),
             r#"namespace FakeStdLib {
                 operation Fake() : Unit {}
-                operation FakeWithParam(x: Int) : Unit {}
+                operation FakeWithParam(x : Int) : Unit {}
```

Comment:
I'm open to updating them, but I like having the preface of `Fake` because it tells me in-line exactly where it came from, instead of having to remember `WithParam` comes from the `FakeStdLib` namespace. It is a little thing, but I find it useful.